docs/en/sdk-manual/wechat.md

```diff
@@ -0,0 +1,323 @@
+| _profile_set       | when `addUserAttributes()` or `setUserId()` API is invoked                                                                                                                                                                                                                                                                |                                                              |                                                              |
+| _mp_share          | when user shares the mini program to others                                                                                                                                                                                                                                                                               |                                                              |                                                              |
+| _mp_favorite       | when user adds the mini program to favorites                                                                                                                                                                                                                                                                              |                                                              |                                                              |
+| _clickstream_error | when event_name is invalid or attribute name/value is invalid                                                                                                                                                                                                                                                             | 1. *_error_code* <br/>2. _*error_message*                    |                                                              |
+
+### Session definition
+
+In Clickstream WeChat Miniprogram SDK, we do not set duration max limit for a session. If the user opens the mini program within a certain time period after they quit the mini program, we still consider it being in the same session.
+
+1. **_session_start**: When the mini program is launched for the first time, or the mini program is open to the foreground and the time between the last exit exceeds `session_time_out` period.
+
+1. **_session_duration**: We calculate the `_session_duration` based on the current event creation timestamp and the session's `_session_start_timestamp`, this attribute will be added in every event during the session.
+
+1. **session_time_out**: 30 minutes by default, which can be customized via the configuration API.
+
+1. **_session_number**: The total number of distinct sessions (by session id), this attribute will be added in every event's attribute object.
+
```

Comment:
Item is not correctly numbered.

Reply:
It works in Markdown and it is rendered correctly in local.